diff --git a/node_modules/@zenfs/core/dist/index.js b/node_modules/@zenfs/core/dist/index.js
index c4121cc..78c47ab 100644
--- a/node_modules/@zenfs/core/dist/index.js
+++ b/node_modules/@zenfs/core/dist/index.js
@@ -17,4 +17,4 @@ export default fs;
 export * from './node/compat.js';
 export * as vfs from './vfs/index.js';
 import $pkg from '../package.json' with { type: 'json' };
-globalThis.__zenfs__ = Object.assign(Object.create(fs), { _version: $pkg.version });
+globalThis.__zenfs__ = Object.assign(Object.create(null), fs, { _version: $pkg.version });
